new/PRS14518Gresley.xml

```diff
@@ -0,0 +1,56 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS14518Gresley" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>M. Gresley</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="CH"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <langUsage>
+                <language ident="en">English</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2024-07-25">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="2">
+                    <persName><forename>M.</forename> <surname>Gresley</surname></persName>
+                    <floruit notBefore="1900" notAfter="1978">20th century</floruit>
```

Comment:
Indicate it exactly: 20th century, but not after 1978 (see below).

Reply:
I think it is better to change to 1900 to 1999. We do not know, whether this was a young or an old Miss, when she bequeathed the scroll to the British Museum.

Reply:
BUT we have the catalogue as the terminus ante quem

Reply:
Is it sure that it was post 1900? 
Gresley are a known family https://en.wikipedia.org/wiki/Gresley_baronets and a certain Miss M. Gresley from the family published in 1860. https://research-repository.st-andrews.ac.uk/bitstream/handle/10023/25063/Luxford_2018_ManuscriptsEandW_Ridware_AAM.pdf

Reply:
It is possible that this is the family. However, the manuscript was given to the BL in 1944. It is unlikely that this is the same Miss M. Gresley who published in 1860. 

It is always problematic to add a floruit when little is known about a person. This person could be a very young woman when she presents the manuscript, which she may have inherited, but she could also be a rather old woman, although the title 'Miss' suggests that she was not married in that year. In her case, the floruit (= the main period of her activity) could be either mostly before 1944 or after 1944. The date of the catalogue should not be decisive here.